### PR TITLE
Fix up some of the recent newsletters

### PR DESCRIPTION
**Pull request by bradystroud**
- Affected routes: 

- Fixed #{{ ISSUE NUMBER }}

- [ ] If adding a new page, I have followed the [📃 New Webpage](https://github.com/SSWConsulting/SSW.Website/issues/new?assignees=&labels=&projects=&template=new_webpage.yml&title=%F0%9F%93%84+%7B%7B+TITLE+%7D%7D+) issue template

- [ ] Include done video or screenshots

### CODE REVIEW

File: public/images/newsletter-uploads/_2023_11__Effective Feedback When and How to Offer It__.html
Comment (Calinator444) [inline public/images/newsletter-uploads/_2023_11__Effective Feedback When and How to Offer It__.html:1]: Why was this file deleted? Is it still in the JSON file?

File: public/images/newsletter-uploads/2024/_2024_08__How to Modernize an Enterprise Application to dotnet 8 and Lose 75 of the Code__.html
Comment (Calinator444) [inline public/images/newsletter-uploads/2024/_2024_08__How to Modernize an Enterprise Application to dotnet 8 and Lose 75 of the Code__.html:1]: Suggestion - I'd rename this to "How to Modernize an Enterprise Application to dotnet 8 and Lose 75 **Percent** of the Code"

File: content/newsletters/_2024.json [hunk 1]
```diff
@@ -30,6 +25,31 @@
...
+      "description": "Up and running with 🦙TinaCMS in 15 minutes - No prob-llamas!"
+    },
+    {
+      "month": 7,
+      "file": "/images/newsletter-uploads/2024/_2024_07__Showcasing TinaCMS 🦙 at Europe's No1 Developer Conference__.html",
+      "description": "Showcasing TinaCMS 🦙 at Europe's #1 Developer Conference"
+    },
+    {
+      "month": 8,
+      "file": "/images/newsletter-uploads/2024/_2024_08__How to Modernize an Enterprise Application to dotnet 8 and Lose 75 of the Code__.html",
```
Comment (Calinator444) [inline content/newsletters/_2024.json]:
```suggestion
      "file": "/images/newsletter-uploads/2024/_2024_08__How to Modernize an Enterprise Application to dotnet 8 and Lose 75 Percent of the Code__.html",
```

In line with my other [suggestion](https://github.com/SSWConsulting/SSW.Website/pull/3061#discussion_r1741447268)

File: public/images/newsletter-uploads/_2023_11__Effective Feedback When and How to Offer It__.html
Comment (bradystroud) [reply public/images/newsletter-uploads/_2023_11__Effective Feedback When and How to Offer It__.html:1]: duplicate

**Review by Calinator444 — APPROVED** [review]
